core/dbt/common/utils/jinja.py

```diff
@@ -0,0 +1,33 @@
+from dbt.common.exceptions import DbtInternalError
+
+
+MACRO_PREFIX = "dbt_macro__"
```

Comment:
Should this just be folded into `clients.jinja`? It seems like we're defining app logic in a utils package which exceeds the implied scope of a "utility" module.

Reply:
Created a follow-on issue for further refactoring in dbt/common: https://github.com/dbt-labs/dbt-core/issues/9206